common-api/src/main/scala/pl/touk/nussknacker/engine/api/Comment.scala

```diff
@@ -6,6 +6,13 @@ final case class Comment(content: String) extends AnyVal {
   override def toString: String = content
 }
 
+final case class NonEmptyComment private (content: String)
```

Comment:
WDYT about using Refined lib and NonEmptyString as content in the `Comment` case class.
I don't think I understand why we need to have Comment and NonEmptyComment? Do we need to distinguish these cases in business logic? Or an empty content is just no comment at all? (so it could be expressed as Option[Comment] obviously)

Reply:
For now I did it without refined (it is not present in Nu right now, definitely should be added in future). I removed NonEmptyComment and Comment now has private constructor and create method which validates string value.

One usage in Dto class was problematic, replaced with raw string in dto.